go/logictest/runner.go

```diff
@@ -239,15 +247,44 @@ func runTestFile(harness Harness, file string) {
 	}
 
 	for _, record := range testRecords {
-		_, _, cont, _ := executeRecord(harness, record)
+		_, _, cont, _ := executeRecord(harness, record, false)
+		if !cont {
+			break
+		}
+	}
+}
+
+func runTestFileWithTestTimes(harness Harness, file string) {
+	currTestFile = file
+
+	err := harness.Init()
+	if err != nil {
+		panic(err)
+	}
+
+	testRecords, err := parser.ParseTestFile(file)
+	if err != nil {
+		panic(err)
+	}
+
+	for _, record := range testRecords {
+		_, _, cont, _ := executeRecord(harness, record, true)
 		if !cont {
 			break
 		}
 	}
 }
 
+
 // Executes a single record and returns whether execution of records should continue
-func executeRecord(harness Harness, record *parser.Record) (schema string, results []string, cont bool, err error) {
+func executeRecord(harness Harness, record *parser.Record, addDurationToLog bool) (schema string, results []string, cont bool, err error) {
+	var start time.Time
```

Comment:
Recommend moving this to a global var like the other ones that end up logged, call it startTime